Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
Yes I made those comments and then corrected myself further in the thread

On Mon, Dec 21, 2015, 23:45 Tom Spilman <notifications@github.com> wrote:

> In Build/Projects/FrameworkReferences.definition
> <https://github.com/mono/MonoGame/pull/4339#discussion_r48142241>:
>
> > @@ -25,6 +29,7 @@
> >          Name="OpenTK"
> >          Path="ThirdParty/Dependencies/OpenTK.dll" />
> >      </Service>
> > +    <Reference Include="MonoGame.Framework.Core.Include" />
>
> because the Linux platform isn't using PCLs here.
>
> Oh... well is that why @DDReaper <https://github.com/DDReaper> is putting <Reference
> Include="MonoGame.Framework.Core" /> into platform specific sections?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/MonoGame/pull/4339/files#r48142241>.
>